acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -49,5 +49,32 @@ def gem_command(host, type='aio')
       end
       module_function :gem_command
     end
+
+    module ManifestUtils
```

Comment:
Are these used anywhere? In the 5.5.x branch, it looks like they were only added for cron tests, which have since been removed. If they aren't used, we shouldn't bring them forward
```
acceptance/lib/puppet/acceptance/common_utils.rb:    module ManifestUtils
acceptance/tests/resource/cron/should_not_overwrite_crontab_file_on_file_read_error.rb:  extend Puppet::Acceptance::ManifestUtils
acceptance/tests/resource/cron/should_only_fail_associated_resources_on_file_read_error.rb:  extend Puppet::Acceptance::ManifestUtils
```
from a checkout of this merge up:
```
[0] Melissa@bubba:/Users/Melissa/puppet:(6.0.x)$ git grep ManifestUtils
acceptance/lib/puppet/acceptance/common_utils.rb:    module ManifestUtils
```

Reply:
They will likely be used in future acceptance tests so it would be good to keep that. The `cron_manifest` routine can be removed, but I think keeping it there might be a good idea to avoid annoying merge conflicts in the future (e.g. in case an `exec_manifest` definition is added in 5.5.x below that).